src/Magnum/CMakeLists.txt

```diff
@@ -185,7 +185,9 @@ endif()
 target_include_directories(Magnum PUBLIC
     ${PROJECT_SOURCE_DIR}/src
     ${PROJECT_BINARY_DIR}/src)
-if(BUILD_DEPRECATED AND TARGET_GL) # TODO: remove once compat gets dropped
+if(BUILD_DEPRECATED) # TODO: remove once compat gets dropped
+    # Some deprecated APIs use headers (but not externally defined symbols)
+    # from the GL library, link those includes as well
```

Comment:
In many places I used the following preprocessor statement:

```cpp
#if defined(MAGNUM_BUILD_DEPRECATED) && defined(MAGNUM_TARGET_GL)
```

but there are apparently cases that I missed, having just `#ifdef MAGNUM_BUILD_DEPRECATED`. Sorry.

... so, instead of installing the GL stuff even when the user *explicitly doesn't want to*, the deprecated stuff depending on GL should not be compiled at all when `TARGET_GL` is `OFF`. So all these CMake patches that add GL stuff should be removed and replaced by preprocessor patches that remove GL stuff.

I have no easy way of testing this myself without creating a barebones VM from scratch, so I'm kinda depending on you to find and fix all those cases :) Thank you :+1:

Reply:
Alright, that will require time that I currently don't have, though. I will have the time to work through this as of the 9th of July, I hope that is okay.